pkg/metrics/metrics.go

```diff
@@ -0,0 +1,104 @@
+package metrics
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	OCMAgentTotalRequests = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "ocm_agent_total_requests",
+			Help: "A count of total requests to ocm agent service",
+		}, []string{})
+
+	OCMAgentByServiceRequests = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "ocm_agent_service_requests",
```

Comment:
given that I know the ocm-agent architecture, this metric is not entirely clear. 

Is it:

* a request to ocm-agent ?
* a request api.openshift.com?
* both?

Maybe we could hint to the operator reviewing these metrics which request this is?

Reply:
This is the counter for the request to ocm-agent, but for different paths/endpoints.
Eg, we are supporting alertmanager as an endpoint for ocm-agent for now.
The counter here will count all the requests to the alertmanager endpoints. 
And if we are going to support the other endpoints in future. There will be a different counter for them. 
It just be used for statistics.